.github/workflows/move_tests.yaml

```diff
@@ -0,0 +1,18 @@
+name: "Move Tests"
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - testnet
+      - mainnet
+
+jobs:
+  move-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install dev dependencies.
+        run: ./move/sh_scripts/dev_setup.sh
+      - name: Runs move tests.
+        run: ./move/sh_scripts/move_tests.sh
```

Comment:
Noob question: what's this for?

Reply:
Also what are the .idea files?

Reply:
.idea files are intellij config, i shuold have ignored them

Reply:
this ci first install aptos cli then run the script which runs move unit test